src/Samples/Sample.AspNetCore/Controllers/VerifyController.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Sample.AspNetCore.Data;
+using Sample.AspNetCore.Models;
+using SwedbankPay.Sdk;
+using SwedbankPay.Sdk.Payments;
+using SwedbankPay.Sdk.Payments.CardPayments;
+
+namespace Sample.AspNetCore.Controllers
+{
+    public class VerifyController : Controller
+    {
+        private readonly ISwedbankPayClient swedbankPayClient;
+        private readonly Cart cartService;
+        private readonly PayeeInfoConfig payeeInfoOptions;
+        private readonly UrlsOptions urls;
+        private bool generatePaymentToken;
+        private bool generateReccurenceToken;
+        private string description;
+        private string userAgent;
+        private readonly StoreDbContext context;
+        private readonly SwedbankPayConnectionSettings swedbankPayConnectionSettings;
+
+        public VerifyController(IOptionsSnapshot<PayeeInfoConfig> payeeInfoOptionsAccessor,
+                              IOptionsSnapshot<UrlsOptions> urlsAccessor,
+                              Cart cartService,
+                              ISwedbankPayClient swedbankPayClient, StoreDbContext context, SwedbankPayConnectionSettings swedbankPayConnectionSettings)
+        {
+            this.payeeInfoOptions = payeeInfoOptionsAccessor.Value;
+            this.urls = urlsAccessor.Value;
+            this.cartService = cartService;
+            this.swedbankPayClient = swedbankPayClient;
+            this.context = context;
+            this.swedbankPayConnectionSettings = swedbankPayConnectionSettings;
+        }
+
+        public IActionResult Index()
+        {
+            return View();
+        }
+
+        public IActionResult Privacy()
+        {
+            return View();
+        }
+
+        public async Task<CardVerify> InitiateVerify()
```

Comment:
```suggestion
        private async Task<CardVerify> InitiateVerify()
```
Make method private and move to the bottom of the method list along with other private methods.